### DISCUSSION TITLE
Cease Community RPC Node Incentives and End the Infra Program

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#173] [General]
## Background

Recent tests and experiments from DevOps indicate that having a collator node also serve as an RPC node could significantly degrade block production performance. Additionally, the community RPC incentive program has proven ineffective.

Reference: https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x624dc48479f8f87d28c68e9ae4fb4d5aaf5cb2452b14ac32a3ab0791e8c018a3

## Proposal

Cease Community RPC Node Incentives: Starting in September, all community RPC node incentives will be discontinued. The budget originally allocated for these incentives will be redirected to establish dedicated RPC nodes to enhance overall performance and stability. cc @fewensa @casinoyoda please notify [current community rpc providers](https://github.com/dcdao/community-rpc/tree/main/provider) in the program in advance.

End the Infra Program: Beginning in Q4(October), DCDAO will terminate the Infra Program. The nodes currently maintained under this program will be handed over to Itering or a third party for maintenance after discussions with Itering. If additional resources are needed, separate applications can be submitted to the Treasury. Since the current Infra Program is proposed by @fewensa from Itering, handling and moving this program to be managed by Itering might be a simpler choice

**Comment by nblogist** [comment]
> Additionally, the community RPC incentive program has proven ineffective.

Was this advertised enough? If so i think its better to bring it on when we have more devs- should help having more exposure.

**Comment by hackfisher** [comment]
community rpc will be replace by alternative solutions like https://subquery.network/rpc

Existing rpc nodes providers can consider and join the new alternative way of contributing.
